spectrocloud/resource_cluster_eks.go

```diff
@@ -560,27 +560,29 @@ func resourceClusterEksUpdate(ctx context.Context, d *schema.ResourceData, m int
 
 		for _, mp := range ns {
 			machinePoolResource := mp.(map[string]interface{})
-			name := machinePoolResource["name"].(string)
-			hash := resourceMachinePoolEksHash(machinePoolResource)
-
-			machinePool := toMachinePoolEks(machinePoolResource)
-
-			var err error
-			if oldMachinePool, ok := osMap[name]; !ok {
-				log.Printf("Create machine pool %s", name)
-				err = c.CreateMachinePoolEks(cloudConfigId, machinePool)
-			} else if hash != resourceMachinePoolEksHash(oldMachinePool) {
-				// TODO
-				log.Printf("Change in machine pool %s", name)
-				err = c.UpdateMachinePoolEks(cloudConfigId, machinePool)
-			}
+			if machinePoolResource["name"].(string) != "" {
```

Comment:
```suggestion
                        // since known issue in TF SDK: https://github.com/hashicorp/terraform-plugin-sdk/issues/588
			if machinePoolResource["name"].(string) != "" {
```